app/internal/rest.py

```diff
@@ -56,4 +57,30 @@ def handler(generic):
     else:
         response_body = {generic: request.json}
 
+    # TODO - will be addressed in TEAM-1487
```

Comment:
This should be more explicit: 
```suggestion
    # TODO - CodeQL issue with this line will be addressed in TEAM-1487
```